packages/react/webpack.config.js

```diff
@@ -0,0 +1,65 @@
+let path = require('path');
+const resources = require('../../scripts/webpack/webpack-resources');
+const ManifestServicePlugin = require('@uifabric/webpack-utils/lib/ManifestServicePlugin');
+
+const BUNDLE_NAME = 'fluent-ui-react';
```

Comment:
Should this be `fluentui-react` to more closely match the package name?

Reply:
interesting suggestion.